src/server/game/Spells/SpellInfo.h

```diff
@@ -201,6 +201,7 @@ enum SpellCustomAttributes
     SPELL_ATTR0_CU_POSITIVE_EFF0                 = 0x02000000,
     SPELL_ATTR0_CU_POSITIVE_EFF1                 = 0x04000000,
     SPELL_ATTR0_CU_POSITIVE_EFF2                 = 0x08000000,
+    SPELL_ATTR0_CU_REJECT_AURA_SAVING            = 0x10000000,
```

Comment:
I would prefer to have them together. So pls move SPELL_ATTR0_CU_FORCE_AURA_SAVING to be at 0x20000000

Reply:
Applied